src/main/g8/build.sbt

```diff
@@ -1,11 +1,5 @@
-
-val defaultDottyVersion = "0.1.2-RC1"
-val defaultScala2Version = "2.12.2"
-
-val useLatestDottyNightly = settingKey[Boolean]("Use the latest Dotty nightly build.")
```

Comment:
This template is not an appropriate place to test nightly builds, this is not something that should be part of the build of everyone that tries to use dotty. If you really want to do so, you could inject a scalaVersion := latestDottyNightly... when running the tests.